web/packages/teleport/src/Discover/ConnectMyComputer/SetupConnect/SetupConnect.tsx

```diff
@@ -39,6 +62,89 @@ export function SetupConnect(props: AgentStepProps) {
     path: Path.ConnectMyComputer,
   });
 
+  const { node, isPolling } = usePollForConnectMyComputerNode({
+    username,
+    clusterId,
+    pingInterval,
+  });
+
+  const [showHint, setShowHint] = useState(false);
+  useEffect(() => {
+    if (isPolling) {
+      const id = window.setTimeout(() => setShowHint(true), showHintTimeout);
+
+      return () => window.clearTimeout(id);
+    }
+  }, [isPolling, showHintTimeout]);
+
+  let hint: JSX.Element;
+  if (showHint && !node) {
```

Comment:
I was mostly just following what was in `web/packages/teleport/src/Discover/Server/DownloadScript/DownloadScript.tsx`. 🫢

Maybe ill' just change the name of the `hint` variable to `pollingStatus`? I feel like I'd rather keep `showHint` to follow the convention from other components, since this variable is used in multiple places to implement some logic.

Reply:
>Maybe ill' just change the name of the hint variable to pollingStatus.

Sounds good.